Cabal/Distribution/Simple/Configure.hs

```diff
@@ -193,7 +194,11 @@ instance Show ConfigStateFileError where
 
 instance Exception ConfigStateFileError
 
-getConfigStateFile :: FilePath -> IO LocalBuildInfo
+-- | Read the LocalBuildInfo file.  Throw an exception if the file is
```

Comment:
Other comments say `'localBuildInfoFile'` instead of `LocalBuildInfo file`. Let's be consistent.